src/compoenents/CustomerMailDetailPannel/YieldHistoryTb.tsx

```diff
@@ -0,0 +1,80 @@
+
+import React, { useEffect, useState } from 'react';
+import { Box, Button, Grid, Paper, Typography } from '@mui/material';
+import '../../assets/styles/yieldHistoryTb.css';
+
+interface HistoryData {
+  name: string;
+  organization: string;
+  estate: string;
+  yieldStressDisease: string;
+}
+
+const YieldHistoryTb: React.FC = () => {
+  const [data, setData] = useState<HistoryData[]>([]);
+
+  useEffect(() => {
+    const dummyData: HistoryData[] = [
+      {
+        name: "John Doe",
+        organization: "AgriCorp",
+        estate: "North Field",
+        yieldStressDisease: "High, Low, Medium",
+      },
+      {
+        name: "Jane Smith",
+        organization: "Farmers Inc.",
+        estate: "West Field",
+        yieldStressDisease: "Medium, High, Low",
+      },
+      {
+        name: "Mike Johnson",
+        organization: "GreenFields",
+        estate: "South Field",
+        yieldStressDisease: "Low, Medium, High",
+      },
+      // dummy data 
+    ];
+
+    // Simulate fetching data from an API
+    setData(dummyData);
+  }, []);
+
+  return (
+    <div className="history-table-container">
+      <h2>Yield History</h2>
```

Comment:
Consider using the MUI Typography component instead of the <h2> tag for Yield History to stay consistent with the library used in the project.